data/data-pipeline/data_pipeline/tests/sources/example/test_etl.py

```diff
@@ -198,9 +186,68 @@ def test_fixtures_contain_shared_tract_ids_base(self, mock_etl, mock_paths):
         else:
             raise NotImplementedError("This geo level not tested yet.")
 
+    def test_sample_data_exists(self):
```

Comment:
the sample data only exists in the `data` folder and there is no secondary `tmp` folder. The `tmp` paths below are the temp files.